client/verta/tests/registry/test_validate_schema.py

```diff
@@ -0,0 +1,122 @@
+from datetime import timedelta
+
+import hypothesis
+import pytest
+from jsonschema import ValidationError
+
+from tests.strategies import generate_object, generate_another_object
+from verta.registry import validate_schema, verify_io
+
+
+class TestValidateSchema:
+    @hypothesis.settings(
+        suppress_health_check=[hypothesis.HealthCheck.function_scoped_fixture],
+        deadline=timedelta(milliseconds=50),
+    )
+    @hypothesis.given(
+        matching_input_value=generate_object(),
+        matching_output_value=generate_another_object(),
+    )
+    def test_validate_schema_allow(
+        self,
+        recwarn,
+        make_model_schema_file,
+        matching_input_value,
+        matching_output_value,
+    ):
+        @validate_schema
+        def predict(self, input):
+            return matching_output_value.dict()
+
+        predict(None, matching_input_value.dict())
```

Comment:
We should either just define a simple class here, or get rid of `self`; `predict(None, ...)` feels kinda janky

Reply:
It looks like a bunch of our other tests do this and I don't reaaaaaally want to deal with it right now. You okay letting this jankiness slide?

Reply:
I tried getting rid of the self and nothing worked 😭

Reply:
Oh. Do we? Welp. I guess it's fine haha